Resources/translations/FOSUserBundle.vi.yml

```diff
@@ -40,20 +40,34 @@ registration:
         user_created: 'Người sử dụng đã được khởi tạo'
     email:
         subject: 'Chào mừng %username%!'
-        message: 'Xin chào %username%! Để hoàn tất kích hoạt tài khoản của bạn - vui lòng truy cập %confirmationUrl% Cám ơn.'
+        message: |
+        Xin chào %username%!
```

Comment:
The indentation is broken here, making the YAML invalid.

Reply:
My bad. Please check [this PR](https://github.com/FriendsOfSymfony/FOSUserBundle/pull/2881)